support/Pipelines/Bbh/Inspiral.py

```diff
@@ -18,6 +18,14 @@
 INSPIRAL_INPUT_FILE_TEMPLATE = Path(__file__).parent / "Inspiral.yaml"
 
 
+def _parse_id_horizons_path(ctx, _, value):
```

Comment:
You can delete this function. I think it's better to have only one place where the default is resolved.